components/backup/base/external-secret.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: CHANGE_ME
+  namespace: openshift-adp
+spec:
+  dataFrom:
+  - extract:
+      conversionStrategy: Default
+      decodingStrategy: None
+      key: CHANGE_ME
+  refreshInterval: 1h
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: appsre-vault
+  target:
+    creationPolicy: Owner
+    deletionPolicy: Delete
+    name: CHANGE_ME
```

Comment:
Same comment as above: since we create one secret per cluster, it think it would be better to have the same name for the secret in all cluster, this will be simpler to configure adp operator.

Reply:
ok same comment as above: the name now the same for all clusters but let's find a better name :) . What is the secret, bucket key? give it a name that describe what is this secret